joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/utils/dependency/GradleDependencies.scala

```diff
@@ -114,16 +114,17 @@ object GradleDependencies {
 
       val hasAndroidPrefixGradleProperty =
         try {
-          val out                      = new ByteArrayOutputStream()
-          val gradlePropertiesTaskName = "properties"
-          connection
-            .newBuild()
-            .forTasks(gradlePropertiesTaskName)
-            .setStandardOutput(out)
-            .run()
-          out.close()
-          val gradleAndroidPropertyPrefix = "android."
-          out.toString.split('\n').filter(_.startsWith(gradleAndroidPropertyPrefix)).nonEmpty
+          Using.resource(new ByteArrayOutputStream()) { out =>
+            val gradlePropertiesTaskName = "properties"
+            connection
+              .newBuild()
+              .forTasks(gradlePropertiesTaskName)
+              .setStandardOutput(out)
+              .run()
+            out.close()
```

Comment:
This is done by `Using.resource` itself.

Reply:
nice catch, thanks!